installer/windows/install.bat

```diff
@@ -0,0 +1,12 @@
+@echo off
+if "%1"=="" (
+    echo "missing required msi path argument"
+    exit /B 1
+)
+
+rem uninstall existing storagenode product
+echo uninstalling storagenode
+msiexec /x %1
```

Comment:
maybe `msiexec /uninstall %1 /qn`? `/uninstall` will be easier to understand and `/qn` will suppress question about uninstalling.

Reply:
I left `/qn` out because my experience was that if the installer wasn't installed it caused the rest of the batch script to fail. It's possible that something else was going on. I'd have to check again.

Reply:
Additionally using `/qn` instead of `/qb` doesn't work unless you're running from an privileged environment and no privilege escalation is prompted.

Reply:
we can leave it like that right now but this probably will need to be resolved later if we will start using this script for automated tests. To run it without any question we need only run console as admin.

Can you just change `/x` to `/uninstall` to be clear what this script is doing?

Reply:
Done.